tests/cypress/e2e/results.cy.js

```diff
@@ -513,17 +513,30 @@ config?.pages?.forEach((page, _i) => {
 						// apply some filter
 						let previousFilterLength;
 
-						cy.get(`${config.selectors.sidebar.facetWrapper}`).each((facet, idx) => {
-							if (idx == 0) {
-								// click on an option in facet and ensure urlManager contains new state
-								const facetListOption = facet.find(config.selectors.sidebar.facetOption)[0];
-								cy.get(facetListOption).click({ force: true });
-								cy.snapController().then(({ store }) => {
-									expect(store.filters.length).to.greaterThan(0);
-									previousFilterLength = store.filters.length;
-								});
-							}
-						});
+						const listFacet = store.facets.filter((facet) => facet.display === 'list')[0];
+							if (!listFacet) this.skip();
+
+							cy.get(`${config.selectors.sidebar.facetWrapper}`).each((facet) => {
+								// find matching facet in dom
+								const title = facet.find(config.selectors.sidebar.facetTitle);
+								if (listFacet.label.trim() === title.text().trim()) {
+									if (listFacet.collapsed) {
+										// toggle visibility if collapsed
+										listFacet.toggleCollapse();
```

Comment:
May want to add a _1ms_ delay after `toggleCollapse()` usage (as seen when testing library components)

```
cy.wait(1);
```